pyproject.toml

```diff
@@ -229,14 +231,14 @@ DEP001 = [
 DEP002 = [
   "setuptools",  # Incorrectly used without declaring in some connectors (e.g. source-shopify)
   "cryptography",  # Constrained as transitive dependency due to a bug in newer versions
-  "google-cloud-secret-manager",  # Deptry can't detect that `google.cloud.secretmanager_v1` uses this package
+  "google-cloud-secret-manager",  # Deptry can't detect that `google.cloud.secretmanager_v1` uses this package - now optional dependency
   "dateparser",  # Used by `source-amazon-seller-partner` for parsing string-formatted dates w/ timezone abbreviations. May add to `AirbyteDateTime`.
   "unidecode",  # IMPORTANT: DO NOT USE. Added to support source-google-sheets migration. This dep is deprecated in favour of `anyascii` and is slated for removal.
 
   # TODO: Remove these dependencies if not needed:
   "avro",  # Only imported in `unit_tests` code
-  "psutil",
-  "rapidfuzz",
+  "psutil",  # Now optional dependency in 'system' extra
+  "rapidfuzz",  # Now optional dependency in 'system' extra
```

Comment:
Unless we know that we need them, let's fully remove psutil and rapidfuzz, rather than making optional.